pom.xml

```diff
@@ -41,10 +41,34 @@
 
     <!-- Scm release info -->
     <scm>
-        <connection>scm:git:git://github.com/rschreijer/lutung.git</connection>
-        <developerConnection>scm:git:git@github.com:rschreijer/lutung.git</developerConnection>
-        <url>http://github.com/rschreijer/lutung</url>
+        <connection>scm:git:git://github.com/konturio/lutung.git</connection>
+        <developerConnection>scm:git:git@github.com:konturio/lutung.git</developerConnection>
+        <url>https://github.com/konturio/lutung</url>
     </scm>
+    <repositories>
+        <repository>
+            <id>kontur</id>
+            <url>https://nexus.kontur.io/repository/common/</url>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+        </repository>
+    </repositories>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>kontur</id>
+            <url>https://nexus.kontur.io/repository/common/</url>
```

Comment:
Make sure this url is open for public

Reply:
PDC Nexus endpoint for Kontur libraries is here:
http://devapps.pdc.org/nexus/content/repositories/kontur-snapshot/
http://devapps.pdc.org/nexus/content/repositories/kontur-release/

Credential has been send to Konstantin separately

Reply:
fixed